pkg/command/domain/human_readable_flags_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package domain
+
+import "testing"
+
+func Test_formatSelectorForPrint(t *testing.T) {
+	tests := []struct {
+		name   string
+		input  string
+		output string
+	}{
+		{"normal case with one selector key value", "selector:\n  key1: value1\n", "key1=value1; "},
```

Comment:
Could we also trim the final `;` + the trailing space ? I think this looks more streamlined

Reply:
You are right. Code changed. Thank you.